genie-agent/src/test/groovy/com/netflix/genie/agent/cli/JobRequestConverterSpec.groovy

```diff
@@ -131,6 +132,35 @@ class JobRequestConverterSpec extends Specification {
         1 * validator.validate(_ as AgentJobRequest) >> Sets.newHashSet()
     }
 
+    @Unroll
+    def "Convert arguments: #providedJobArguments"() {
+
+        setup:
+        ArgumentDelegates.JobRequestArguments jobRequestArgs = Spy(new JobRequestArgumentsImpl(mainCommandArguments))
+        AgentJobRequest jobRequest
+
+        when:
+        jobRequest = converter.agentJobRequestArgsToDTO(jobRequestArgs)
+
+        then:
+        1 * jobRequestArgs.getCommandArguments() >> providedJobArguments
+        1 * validator.validate(_ as AgentJobRequest) >> Sets.newHashSet()
+
+        expect:
+        jobRequest.getCommandArgs() == expectedJobArguments
+
+        where:
+        providedJobArguments | expectedJobArguments
```

Comment:
You might want to run the formatter here or this will change later

Reply:
👍